curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-debugging-by-building-a-random-background-color-changer/663250b42513ef5975599c49.md

```diff
@@ -0,0 +1,121 @@
+---
+id: 663250b42513ef5975599c49
+title: Step 2
+challengeType: 0
+dashedName: step-2
+---
+
+# --description--
+
+Now that the `ReferenceError` is resolved, the console is displaying the correct results for a random number between `0` and `9`. But CamperBot was not expecting to see decimal numbers like these:
+
+```js
+0.015882899879771095
+2.114596286197641
+6.040964780197666
+```
+
+Update the `console` statement to return a whole number between `0` and `9`. 
```

Comment:
```suggestion
Update the `console` statement to print a whole number between `0` and `9`. 
```

Otherwise they might replace the `console.log` with `return` (I tried that...)